dll/win32/shell32/wine/shellord.c

```diff
@@ -358,12 +374,12 @@ int ShellMessageBoxW(
 	    hInstance,hWnd,lpText,lpCaption,uType);
 
 	if (IS_INTRESOURCE(lpCaption))
-	  LoadStringW(hInstance, LOWORD(lpCaption), szTitle, sizeof(szTitle)/sizeof(szTitle[0]));
+	  LoadStringW(hInstance, LOWORD(lpCaption), szTitle, ARRAY_SIZE(szTitle));
```

Comment:
These ARRAY_SIZE changes are because I also synchronized with latest Wine (where the code hasn't changed at all except for these).